addons/website_sale/models/res_config_settings.py

```diff
@@ -84,6 +84,11 @@ class ResConfigSettings(models.TransientModel):
         readonly=False,
         required=True,
     )
+    ecommerce_access = fields.Selection(
+        related="website_id.ecommerce_access",
+        readonly=False,
+        required=True,
```

Comment:
What do you mean?

Reply:
The `related` attribute implies some things, for example the `readonly=True` is implied, and the `string` attribute is copied from the related field. Is the `required` attribute also copied? I.e., can you remove this line without changing the behavior?

Reply:
I tried to set the readonly on website model, but it doesn't work. It must be set on res_config_settings model.

Reply:
I suppose, that since it's not stored field, by default it takes all the properties from the related field. I don't know hot to check it since it's not a stored field.